### DISCUSSION TITLE
SSL?

### DISCUSSION THREAD

**Discussion by J3rr1ck** [abhishekkrthakur/diffuzers#16] [General]
Hey there, are there currently any plans to add an arg for HTTPS through uvicorn?

**Comment by abhishekkrthakur** [comment]
You should already be able to do so using tunneling. I'll take a look if i can make it as an option.

**Reply by J3rr1ck** [reply]
I've also built it into my own fork this morning by adding args to pass to uvicorn, more than happy to upstream after some testing if you'd like :)

**Reply by abhishekkrthakur** [reply]
feel free 🙂
